GCC headers fail to behave correctly in 32-bit environment when _KERNEL (and thus _I32LPx) is defined

--- VirtualBox-5.2.22/include/iprt/types.h.1	2018-12-24 09:50:54.822748208 +0000
+++ VirtualBox-5.2.22/include/iprt/types.h	2018-12-24 09:51:28.248468343 +0000
@@ -175,7 +175,9 @@
 #  undef true
 #  undef bool
 # else
+#if !(defined(_KERNEL) && defined(__i386))
 #  include <stddef.h>
+#endif
 #  include <sys/types.h>
 # endif
 
